doc/operator-capabilities.md

```diff
@@ -119,28 +124,41 @@ The Operator offers one or more of the following lifecycle management features:
 
 5. Does your Operator allow adding/removing read-only slave instances of your Operator?
 
+6. Does your operand have a Liveness probe?
+
+7. Does your operand have a Readiness probe which will fail if any aspect of the operand is not ready? e.g. if the connection to the database fails.
+
+8. Does your operand use a rolling deployment strategy?
+
+9. Does your operator create a PodDsruptionBudget resource for your operand pods?
+
+10. Does your operand have CPU requests and limits set?
+
+
+
 ---
 
 ## Level 4 - Deep Insights
 
-**Monitoring**
+Setup full monitoring and alerting for your operand. All resources such as Prometheus rules (alerts) and Grafana dashboards should be created by the operator when the operand CR is instantiated. The RED method is a good place to start with knowing what metrics to expose.
```

Comment:
Provide reference to the RED method.

Reply:
Any preference on which link to include here? 
https://www.weave.works/blog/the-red-method-key-metrics-for-microservices-architecture/ or https://grafana.com/blog/2018/08/02/the-red-method-how-to-instrument-your-services/

Reply:
I think we would prefer an proper definition somewhere rather than a blog article from practitioners.

Reply:
Thanks @dmesser . I am unable to find any reference outside of these. I can add in a short description here or remove the sentence.

Reply:
Yeah, why not define it in a foot note. If not possible in a concise fashion, probably better to remove it.

Reply:
Updated now @dmesser . Let me know what you think.